freqtrade/freqai/data_drawer.py

```diff
@@ -446,7 +446,9 @@ def save_data(self, model: Any, coin: str, dk: FreqaiDataKitchen) -> None:
             dump(model, save_path / f"{dk.model_filename}_model.joblib")
         elif self.model_type == 'keras':
             model.save(save_path / f"{dk.model_filename}_model.h5")
-        elif 'stable_baselines' in self.model_type or 'sb3_contrib' == self.model_type:
+        elif ('stable_baselines' in self.model_type or
+              'sb3_contrib' == self.model_type or
+              'pytorch' == self.model_type):
```

Comment:
```suggestion
        elif (self.model_type in ["stable_baselines", "sb3_contrib", "pytorch"]):
```

Reply:
this statements are not equivalent.
'stable_baselines' in self.model_type vs self.model_type in "stable_baselines"

Reply:
Please keep the suggested change, but you are correct that we simply need to add a "3" to the end of stable_baselines3